src/controllers/CategoryFacetQueryController.ts

```diff
@@ -42,7 +42,12 @@ export class CategoryFacetQueryController {
     lastQuery.groupBy = [groupByRequest];
     lastQuery.categoryFacets.splice(this.categoryFacet.positionInQuery, 1);
     const results = await this.categoryFacet.queryController.getEndpoint().search(lastQuery);
-    return results.groupByResults[0].values;
+    return results.groupByResults[0].values.sort((firstGroupByValue, secondGroupByValue) => {
```

Comment:
You could extract the compare function and give it a significative name eg `orderByNumberOfResultsThenPathLength`